### DISCUSSION TITLE
How to Add RDLC or SSRS Report in Serenity Framework ?

### DISCUSSION THREAD

**Discussion by mbchandak** [serenity-is/Serenity#5318] [Help]
How to Add RDLC or SSRS Report in Serenity Framework in Visual studio 2019?

**Comment by VictorTomaili** [comment]
I transfer this issue to discussion

**Comment by stixoffire** [comment]
@mbchandak I have some sample code - it is really crude - it works , I used the rdlc file (but you don't have to) you could also store those rdlc's in a database and select them for your reports and then use the rdlc pulled form the database as an in memory component.. 
I know it could stand a lot of modification to better conform to serenity (better coding all the way around) . I used it because I ran reports from MySQL - I used a trick on this)  and also SQL server .
